manim/mobject/geometry.py

```diff
@@ -2378,11 +2379,11 @@ def construct(self):
 
     def __init__(
         self,
-        line1,
-        line2,
-        radius=None,
+        line1: "Line",
+        line2: "Line",
```

Comment:
You should import `Line` under `typing.TYPE_CHECKING`, See https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING for more info.

Reply:
But Line is defined in this file (line 761). So I think this is necessary.

Reply:
But as shown in https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING, it does not need the strings.